spring-petclinic-frontend/README.md

```diff
@@ -0,0 +1,43 @@
+# Frontend for PetClinic project
+
+This module is the frontend part of the PetClinic project, it provides the same features as the `spring-petclinic-api-gateway` module.
+When deploying to Azure Spring Apps enterprise plan, you can use the managed component Spring Cloud Gateway on Azure Spring Apps, which is more efficient and cost-effective. 
+
+## Run locally
+
+Please note that supporting services (Config and Discovery Server) must be started before any other application.
+
+Enter the project root directory and open a terminal to start Config Server:
+
+```shell
+./mvnw spring-boot:run -pl spring-petclinic-config-server
+```
+
+Open a new terminal to start Discovery Server:
+
+```shell
+./mvnw spring-boot:run -pl spring-petclinic-discovery-server
+```
+
+For Customers, Vets, Visits and Gateway services, it's required to enable the Spring profile `deployment`,
+Open new terminal in turn and execute the following commands:
+
+```shell
+./mvnw spring-boot:run -Dspring-boot.run.profiles=default,development -pl spring-petclinic-customers-service
+./mvnw spring-boot:run -Dspring-boot.run.profiles=default,development -pl spring-petclinic-vets-service
+./mvnw spring-boot:run -Dspring-boot.run.profiles=default,development -pl spring-petclinic-visits-service
+./mvnw spring-boot:run -Dspring-boot.run.profiles=default,development -pl spring-petclinic-api-gateway
```

Comment:
Is it possible to run as `./mvnw spring-boot:run -Dspring-boot.run.profiles=default,development -pl spring-petclinic-customers-service,spring-petclinic-vets-service`?

Reply:
Use one project for the parameter `-pl`, the testing is OK.